stdlib/public/core/Character.swift

```diff
@@ -203,7 +202,8 @@ public struct Character :
   @_versioned
   internal init(_largeRepresentationString s: String) {
     if let native = s._core.nativeBuffer,
-       native.start == s._core._baseAddress! {
+      native.start == s._core._baseAddress!,
+      native.usedCount == s._core.count {
       _representation = .large(native._storage)
```

Comment:
I doubt it.  Doing just about anything with that character, including comparing it to another character, requires grapheme breaking again if we don't do this.